carpool-map/js/l.geosearch.provider.openstreetmap.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
+1 minification. 

If no minified version exists, it might be worth using an online uglifier like https://skalman.github.io/UglifyJS-online/ to uglify them and concatenate them into one file.